src/app/lib/widgets/enable-when-source/enable-when-source.component.ts

```diff
@@ -109,15 +119,14 @@ export class EnableWhenSourceComponent extends LfbControlWidgetComponent impleme
     this.formProperty.searchProperty('__$answerType').setValue($event.item.data.type, false);
   }
 
-
   /**
    * Format the input after selection
    * @param item - TreeNode object of the item.
    */
   inputFormatter(item: ITreeNode): string {
     let ret: string;
     if (item && item.data) {
-      ret = item.data.text;
+      ret = Util.getIndexPath(item).join('.') + ' ' + item.data.text;
```

Comment:
We probably need little more distinction between the sequence numbers and original text. I think at minimum a separator like ' - ' needed.